mmdet3d/models/middle_encoders/scn.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+from mmcv.cnn import build_norm_layer
+from torch import nn
+
+from mmdet3d.models.builder import MIDDLE_ENCODERS
+from mmdet3d.ops import spconv
+from mmdet3d.ops.spconv.conv import SparseConv3d, SubMConv3d
+
+
+class SparseBasicBlock(spconv.SparseModule):
```

Comment:
SparseBasicBlock is already implemented [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/ops/sparse_block.py#L67).